### ISSUE TITLE
Form - support for role="group" and role="radiogroup"

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#6097]
https://github.com/patternfly/patternfly/pull/4240

A form group with a single label and multiple form elements (text inputs, checkboxes, etc), should have `role="group"` with `aria-labelledby` the `id` of the `<label>` in that form group.
  * Exception to this is if the elements are a grouping of radio buttons where only one can be selected at a time, use `role="radiogroup"`

cc @jessiehuff

**Comment by jessiehuff** [comment]
I think the question I'd have here is which things should be following this pattern? How is this different from the form sections? Our pattern should probably be consistent across. Are we updating our existing examples to use this same pattern? I think it just requires a little clarity. 🙂

**Comment by mcoker** [comment]
> How is this different from the form sections? Our pattern should probably be consistent across.

Opened this issue to add to form sections - https://github.com/patternfly/patternfly/issues/4276. Feel free to add any additional comments.

> Are we updating our existing examples to use this same pattern?

We updated the examples in https://github.com/patternfly/patternfly/pull/4240, so I would expect the form examples to be updated here, too. WDYT?

**Comment by nicolethoen** [comment]
The react examples need to be updated to match the core examples with the use of role="group" as was implemented in this PR https://github.com/patternfly/patternfly/pull/4240. 

@mcarrano can we add this to a milestone?
